frontend/.env.test

```diff
@@ -0,0 +1 @@
+FRONTEND_API_ENDPOINT='https://stagingapp.signoz.io'
```

Comment:
Rather than using `.evn.test` can we not use `.evn` to read the `FRONTEND_API_ENDPOINT` and not push it here to this public repo?